docs/terraform_loadtest.md

```diff
@@ -32,28 +30,28 @@ In order to start the deployment process, it is required to configure the deploy
 cp config/deployer.sample.json config/deployer.json
 ```
 
-Detailed documentation for the deployer's config can be found [here](deployer_config.md).
+Detailed documentation for the deployer's config can be found [here](deployer_config.md). At least, make sure to set the `SSHPublicKey` to the path of your public key and `MattermostLicenseFile` to the path of an enterprise license.
 
 ## Deployment
 
 ### Setup ssh-agent
 
-For the deployer to work, a [ssh-agent](https://linux.die.net/man/1/ssh-agent) needs to be running and loaded with a private key.
+For the deployer to work, an [ssh-agent](https://linux.die.net/man/1/ssh-agent) needs to be running and loaded with a private key.
 
 ```sh
 eval $(ssh-agent -s)
-ssh-add PATH_TO_KEY
+ssh-add PATH_TO_PRIVATE_KEY
 ```
 
-`PATH_TO_KEY` should be replaced with the path to the matching private key for `SSHPublicKey`, as previously [configured](deployer_config.md).
+`PATH_TO_PRIVATE_KEY` should be replaced with the path to the matching private key for `SSHPublicKey`, as previously [configured](deployer_config.md).
 
 ### Create a new deployment
 
 ```sh
 go run ./cmd/ltctl deployment create
 ```
 
-This command can take several minutes to complete when creating a [full](loadtest_system.md) deployment.
+This command can take several minutes to complete when creating a [full](loadtest_system.md) deployment. By default, the console will keep logging info messages; if it does not, then something's wrong.
```

Comment:
Maybe a full stop instead of semi-colon? But really nitpicking here :P

Reply:
Nitpicking granted :)